src/test/scala/fm/http/TestClientAndServer.scala

```diff
@@ -382,7 +389,15 @@ final class TestClientAndServer extends FunSuite with Matchers with BeforeAndAft
   }
 
   test("Content-Type: latin1 & UTF-8 Data") {
-    getSync("/latin1-header-utf8-data", 200, "Â£") // new String("£".getBytes("UTF-8"), "latin1"))
+    getSync("/latin1-header-utf8-data", 200, "Â£")
```

Comment:
Leave the comment but fix the extra `)`:

```scala
getSync("/latin1-header-utf8-data", 200, "Â£") // new String("£".getBytes("UTF-8"), "latin1")
```

Reply:
Actually should update the comment to `new String("£".getBytes("UTF-8"), "ISO-8859-1")` instead of using `latin1"